pkg/backend/diy/postgres/backend.go

```diff
@@ -66,26 +72,14 @@ func IsPostgresBackendURL(urlstr string) bool {
 
 // New creates a new DIY backend using PostgreSQL as the storage layer.
 func New(ctx context.Context, d diag.Sink, postgresURL string, project *workspace.Project) (diy.Backend, error) {
-	// Register the PostgreSQL bucket provider with the blob.URLMux
-	mux := blob.DefaultURLMux()
-	err := PostgresSchemeMux{}.RegisterBucketMux(mux)
-	if err != nil {
-		return nil, err
-	}
-
+	// The PostgreSQL bucket provider is automatically registered via init()
 	// Configure and initialize the PostgreSQL bucket
 	return diy.New(ctx, d, postgresURL, project)
 }
 
 // Login creates or connects to a DIY backend using PostgreSQL as the storage layer.
 func Login(ctx context.Context, d diag.Sink, postgresURL string, project *workspace.Project) (diy.Backend, error) {
```

Comment:
Same as for New. I think the default diy.Login should work now that the default muxer has Postgres.